config/secrets.yml

```diff
@@ -17,6 +17,9 @@
 
 # Environmental secrets are only available for that specific environment.
 
+defaults: &defaults
```

Comment:
Mmmm I'm not a big fan of defaults for secrets. Check my previous comment about this!